driver/src/contracts/snapp_contract.rs

```diff
@@ -58,6 +62,7 @@ where
     }
 }
 
+#[cfg_attr(test, automock)]
```

Comment:
So, if I understand correctly, this is what generated the`MockSnappContract` that is being used in our tests now. It somehow also magically comes along with a `::default()`. This is really great.

Reply:
yes, since it has a new() method without args it can also implement default